src/bindings/build.py

```diff
@@ -34,6 +35,17 @@
     with open(header, "r") as hfile:
         ffi.cdef(hfile.read())
 
+# we need to easily control a few values for Windows builds
```

Comment:
This comment no longe really makes sense, I think you can remove it and replace with just explaining the sodium static bit.